packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -181,45 +240,62 @@ export const Reports = () => {
-                            )}
-                          </MenuItem>,
-                        ]}
-                      >
-                        <IconButton styleType="borderless">
-                          <SvgMore
-                            style={{
-                              width: "16px",
-                              height: "16px",
-                            }}
-                          />
-                        </IconButton>
-                      </DropdownMenu>
+                    selected={selectedReports.some((r) => report.id === r.id)}
+                    onClick={(e) => {
+                      if (e?.target?.className?.toString().startsWith("rcw-horizontal-tile-container")) {
+                        if (!e.ctrlKey)
+                          setSelectedReports([]);
+
+                        setSelectedReports((sr) =>
+                          sr.some((r) => report.id === r.id)
+                            ? sr.filter(
+                              (r) => report.id !== r.id
+                            )
+                            : [...sr, report]
+                        )
+                      }
+                    }
+                    }
+                    actionGroup={
+                      <div className="rcw-button-container">
+                        <ExtractionStatus
+                          state={extractionStates.get(report.id) ?? ExtractionStates.None}>
+                        </ExtractionStatus>
+                        <DropdownMenu
+                          menuItems={(close: () => void) => [
+                            <MenuItem
+                              key={0}
+                              onClick={() => {
+                                setSelectedReport(report);
+                                setReportsView(ReportsView.MODIFYING);
+                              }}
+                              icon={<SvgEdit />}
+                            >
+                              {ReportsConfigWidget.localization.getLocalizedString(
+                                "ReportsConfigWidget:Modify"
+                              )}
+                            </MenuItem>,
+                            <MenuItem
+                              key={1}
+                              onClick={() => {
+                                setSelectedReport(report);
+                                setShowDeleteModal(true);
+                                close();
+                              }}
+                              icon={<SvgDelete />}
+                            >
+                              {ReportsConfigWidget.localization.getLocalizedString(
+                                "ReportsConfigWidget:Remove"
+                              )}
+                            </MenuItem>,
+                          ]}
+                        >
+                          <IconButton styleType="borderless">
+                            <SvgMore
```

Comment:
Close the element on the same line (move `/>` up).

Reply:
`/>` has been moved up